### PR TITLE
model view group layout

### LINKED ISSUE #1489: Model view group layout

### PR DESCRIPTION
**Pull request by llali**
I added a group container for model view controls. This layout is for grouping a set of controls which has a 5 px padding and can used for assigning controls to a radio button. The group can have a header and can be disabled/ enabled as a group

fixes #1489

image

### CODE REVIEW

**Comment by AbbiePetcht** [comment]
Can I see the screenshot?

File: src/sql/parts/modelComponents/inputbox.component.ts [hunk 1]
```diff
@@ -13,7 +13,8 @@ import * as sqlops from 'sqlops';
 import { ComponentBase } from 'sql/parts/modelComponents/componentBase';
 import { IComponent, IComponentDescriptor, IModelStore, ComponentEventType } from 'sql/parts/modelComponents/interfaces';
 
-import { InputBox, IInputOptions, MessageType } from 'vs/base/browser/ui/inputbox/inputBox';
+import { InputBox } from 'sql/base/browser/ui/inputbox/inputBox';
+import { IInputOptions, MessageType } from 'vs/base/browser/ui/inputbox/inputBox';
```
Comment (AbbiePetcht) [inline src/sql/parts/modelComponents/inputbox.component.ts]: revert this

Comment (llali) [reply src/sql/parts/modelComponents/inputbox.component.ts]: why? I wanted to use the sql input box

---
In reply to: [191550709](https://github.com/Microsoft/sqlopsstudio/pull/1508#discussion_r191550709) [](ancestors = 191550709)

Comment (AbbiePetcht) [reply src/sql/parts/modelComponents/inputbox.component.ts]: oh I didn't see it's from sql

---
In reply to: [191554363](https://github.com/Microsoft/sqlopsstudio/pull/1508#discussion_r191554363) [](ancestors = 191554363,191550709)

**Pull request merged**